tests/chainer_tests/functions_tests/pooling_tests/test_max_pooling_nd.py

```diff
@@ -15,7 +15,7 @@
 from chainer.testing import attr
 from chainer.testing import condition
 from chainer.utils import conv
-import pooling_nd_helper
+from tests.chainer_tests.functions_tests.pooling_tests import pooling_nd_helper
```

Comment:
This test passed on Windows with 'from chainer_tests.functions_tests.pooling_tests import pooling_nd_helper'.

Reply:
Thanks, fixed.